### DISCUSSION TITLE
我需要直接调用QQ频道的机器人进行发送消息，我应该怎么建立Group类来发送信息？我看simbot3的文档找不到对应的教程

### DISCUSSION THREAD

**Discussion by dzbxln** [simple-robot/Discussions#175] [Q&A] [answered]
### 涉及的编程语言

Java

### 涉及的组件库

QQ频道

### 疑问描述

如题

**Comment by ForteScarlet** [comment, marked as answer]
参考 #74 ，意思大差不差，把里面的 `MiraiXxx` 改成QQ频道的 `QQGuildXxx` 就行了，比如 `QQGuildBotManager` 和 `QGBot `。
以及，在QQ频道中 `Group` 是无效的，频道对应的应当是 `Guild` 和其中的 `Channel`。
